dice_rolling_simulator.py

```diff
@@ -1,98 +1,77 @@
 #Made on May 27th, 2017
 #Made by SlimxShadyx
+#Editted by CaptMcTavish, June 17th, 2017
 
 #Dice Rolling Simulator
 
 import random
 
-#These variables are used for user input and while loop checking.
-correct_word = False
-dice_checker = False
-dicer = False
-roller_loop = False
+global user_exit_checker
+user_exit_checker="exit"
 
-#Checking the user input to start the program.
-while correct_word == False:
+def start():
+    print "Welcome to dice rolling simulator: \nPress Enter to proceed"
+    raw_input(">")
 
-    user_input_raw = raw_input("\r\nWelcome to the Dice Rolling Simulator! We currently support 6, 8, and 12 sided die! \
-    Type [start] to begin!\r\n?>")
+    result()
 
-    #Converting the user input to lower case.
-    user_input = (user_input_raw.lower())
+def bye():
+    print "Thanks for using the Dice Rolling Simulator! Have a great day! =)"
 
-    if user_input == 'start':
-        correct_word = True
-    
-    else:
-        print "Please type [start] to begin!\r\n"
-
-#Main program loop. Exiting this, exits the program.
-while roller_loop == False:
-
-    #Second While loop to ask the user for the certain die they want.
-    while dice_checker == False:
-        user_dice_chooser = raw_input("\r\nGreat! Begin by choosing a die! [6] [8] [10]\r\n?>")
-
-        user_dice_chooser = int(user_dice_chooser)
-
-        if user_dice_chooser == 6:
-            dice_checker = True
+def result():
 
-        elif user_dice_chooser == 8:
-            dice_checker = True
+    user_dice_chooser
```

Comment:
`user_dice_chooser` is referenced before assignment
Remove this line or comment it.